package/doc/sphinx/source/documentation_pages/loading_files.rst

```diff
@@ -0,0 +1,140 @@
+##################
+Loading your data
+##################
+
+   
+Basic Usage
+===========
+
+
+MDAnalysis aims to read any and all molecular simulation data,
+from whatever files you provide to it.
+This is done via the `Universe` object,
```

Comment:
Are single backtics allowed here? Does napoleon generate links to Universe? Otherwise I suggest using 
```rest
:class:`~MDAnalysis.groups.universe.Universe` 
```